en/intro-concepts.md

```diff
@@ -304,6 +304,10 @@ Deploy.
 Allocates (reserves) a node to the MAAS user performing the action (and
 currently logged in). Changes a node's status from 'Ready' to 'Allocated'.
 
+With the CLI, it is necessary to perform this action prior to deploying. With
+the web UI it is done automatically for the user. Acquiring in the web UI is
+used for machine reservation.
```

Comment:
I know this is a design issue, but I find myself wondering what the difference is between a node being reserved with 'acquire' on the CLI and a machine being reserved from the web UI.